netbox_diode_plugin/api/views.py

```diff
@@ -49,7 +49,11 @@ def _get_lookups(self, object_type_model: str) -> tuple:
 
         """
         if "'ipam.models.ip.ipaddress'" in object_type_model:
-            return "interface", "interface__device", "interface__device__site"
+            return (
+                "assigned_object",
+                "assigned_object__device",
+                "assigned_object__device__site",
+            )
```

Comment:
So the issue was that `prefetch_related`  was failing due to invalid args. It worked before because this code was not being properly called, then I've fixed it with `if "'ipam.models.ip.ipaddress'" in object_type_model:` when I added unit tests.